test/swar/BasicOperations.cpp

```diff
@@ -41,6 +41,15 @@ static_assert(
     multiplication_OverflowUnsafe_SpecificBitCount<3>(Micand, Mplier).value()
 );
 
+TEST_CASE("Jamie's totally working exponentiation :D") {
+    constexpr auto base     = SWAR<8, u32>{0x02'00'05'06}; //   2 | 0 |  5 |   6
+    constexpr auto exponent = SWAR<8, u32>{0x07'00'02'03}; //   7 | 0 |  2 |   3
```

Comment:
Among the purposes of tests is to help readers see things used, so they can mimic what they see until they develop more real understanding.
Let's not use confusing examples like 0^0 which is undefined.